pennylane/ops/qubit/__init__.py

```diff
@@ -41,7 +41,7 @@
 from .state_preparation import *
 from .special_unitary import SpecialUnitary
 from .hamiltonian import Hamiltonian
-from ..identity import Identity
+from ..identity import *
```

Comment:
it's generally preferred to avoid `*`-imports in Python, but we already do it sometimes so it's not the end of the world. Reason (on master, so no GlobalPhase but the point stands):
```pycon
>>> dir_old = dir()
>>> from pennylane.ops.identity import *
>>> dir_new = dir()
>>> set(dir_new) - set(dir_old)
{'Operation', 'lru_cache', 'Identity', 'CVObservable', 'AnyWires', 'sparse', 'dir_old'}
```
from the above list (`dir_old` aside), we only wanted `Identity`. I'm guessing you did this because it caused you a headache when it wasn't properly imported the first time around, but... that's PennyLane development! Not sure what's best so if this sucked enough as a developer, then I'd say `*`-imports are worthwhile to improve that experience

Reply:
changed to `import Identity, GlobalPhase`